app/src/pages/inside/testCaseLibraryPage/createTestCaseModal/selectors.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { formValueSelector } from 'redux-form';
+import type { AppState } from 'types/store';
+import { ManualScenarioType } from '../types';
+
+export const manualScenarioTypeSelector = (formName: string) => (state: AppState) => {
+  const formSelector = formValueSelector(formName);
+  return formSelector(state, 'manualScenarioType') as ManualScenarioType;
+};
+
+export const stepsDataSelector = (formName: string) => (state: AppState) => {
+  const formSelector = formValueSelector(formName);
+  return formSelector(state, 'steps') as Record<string, unknown> | undefined;
```

Comment:
Can we type the steps similar to how you did for manual scenarios?

Reply:
done